### ISSUE TITLE
Ship Data text field should stay in upper left of map

### ISSUE THREAD

**Issue by jpszabo** [leighgarbs/cs420-game#77]
When the window is resized, the ship data (and the currently being implemented game clock) stay in their respective window positions but should probably stay in their default positions on the map (i.e. upper left and upper right corners).

**Comment by leighgarbs** [comment]
I actually thought this was okay when I implemented it.  If it's off the map then it's not blocking the player's view of the map, so I figured if it moved with the window instead of the map that was fine.

**Comment by jpszabo** [comment]
Fair enough. I don't think it looks as good with the game clock because I
just set the x coordinate to be 600. I think I can probably do that smarter.

**Comment by leighgarbs** [comment]
Of course if we want this text to stay over the map we can do that.  Maybe Christine or Justin have an opinion.

**Comment by mrfievel** [comment]
Now that I fixed the game timer this looks pretty weird, since it moves but the timer doesn't.  Not sure how to fix it.
